src/create/index.js

```diff
@@ -28,20 +30,29 @@ module.exports = function generate(params, callback) {
     function _arcFileValid(arc, callback) {
       validate(arc, callback)
     },
+    function _beforeCreate(arc, callback) {
```

Comment:
I think we should revisit this whole file and clean it up to be more like the new deploy index, but I didn't think this should be in the same PR.